src/app/[year]/submission/components/formStep1.tsx

```diff
@@ -0,0 +1,241 @@
+                    const val = e.target.value;
+                    field.onChange(val);
+                  }}
+                  
+                />
+              </FormControl>
+              <FormMessage />
+            </FormItem>
+          )}
+        />
+        <div className="m-4" />
+        <FormField
+          control={form.control}
+          name="description"
+          render={({ field }) => (
+            <FormItem className="w-full">
+  <FormLabel>Description*</FormLabel>
+              <FormControl>
+                <Textarea
+                  placeholder="Enter a short description"
+                  value={field.value}
+                  onChange={(e) => {
+                    const val = e.target.value;
+                    field.onChange(val);
+                  }}
+                  
+                />
+              </FormControl>
+              <FormMessage />
+            </FormItem>
+          )}
+        />
+        <div className="mt-6">
+        <FormField
+          control={form.control}
+          name="photos"
+          render={({ field }) => (
+            <FormItem className="w-full">
+              <FormLabel>Upload Image:</FormLabel>
+              <FormControl>
+                {}
+                <div>
+                  <input
+                    ref={inputRef}
+                    type="file"
+                    multiple
+                    accept="image/png,image/jpeg,image/webp,application/pdf"
+                    className="block w-full text-sm text-zinc-700 file:mr-3 file:rounded-md file:border-0 file:bg-indigo-600 file:px-3 file:py-2 file:text-white hover:file:bg-indigo-700"
+                    disabled={isUploading}
+                    onChange={async (e) => {
+                      setError("");
+                      const files = Array.from(e.target.files || []);
+                      if (files.length === 0) {
+                        return;
+                      }
+
+                      // Client-side guards
+                      for (const file of files) {
+                        if (!ALLOWED.includes(file.type as AllowedMime)) {
+                          setError(
+                            "Unsupported file type. Use PNG, JPG, WEBP, or PDF."
+                          );
+                          if (inputRef.current) inputRef.current.value = "";
+                          return;
+                        }
+                        if (file.size > MAX_BYTES) {
+                          setError("File too large. Max 5MB.");
+                          if (inputRef.current) inputRef.current.value = "";
+                          return;
+                        }
+                      }
+
+                      const current = Array.isArray(field.value)
+                        ? field.value
+                        : field.value
+                        ? [field.value]
+                        : [];
+                      const remaining = 5 - current.length;
```

Comment:
```suggestion
                      const totalImages = 20;
                      const remaining = totalImages - current.length;
```

better to avoid hardcoding and use a more descriptive variable